R/utilities-label.R

```diff
@@ -94,3 +94,22 @@ getLabelWithUnit <- function(label, unit = NULL) {
     return(paste0(label, " [", unit, "]"))
   }
 }
+
+#' Sanitize Label Text
+#' @description
+#' ggtext does not allow certain characters that can be converted to html
+#  tags but that are not supported
```

Comment:
Add that the function actually removes these characters. Also, please add a note to NEWS as this is a new behavior.